tests/eos-test-utils.c

```diff
@@ -28,10 +28,16 @@
 #include <ostree.h>
 #include <string.h>
 
+#include <libeos-updater-util/util.h>
+
 #ifndef GPG_BINARY
 #error "GPG_BINARY is not defined"
 #endif
 
+#ifndef FLATPAK_BINARY
+#error FLATPAK_BINARY is not defined
```

Comment:
Can this actually happen in practice?

Reply:
I'm not sure - I'm essentially just following the convention from the other definitions in eos-test-utils here.